src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx

```diff
@@ -0,0 +1,35 @@
+---
+subject: Android agent
+releaseDate: '2023-09-21'
+version: 7.1.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.1.0.zip'
+---
+
+<Callout variant="important">
+  Welcome to the New Relic Android agent version 7.x. This major release breaks compatibility with versions of 
+  * Gradle 6.x
+  * Android Gradle Plugin (AGP) 4.x
+
+  Android agent 7.x is increasing the minimum Gradle version to 7.1. Customers will be required to upgrade to Android Gradle Plugin (AGP) 7.0.0 and higher.
+</Callout>
+
+## New
+
+* Adds support for DexGuard 9+ with Gradle 8/AGP 8
+* Adds unity support with custom exceptions
+* Adds map parameters for both custom and auto HTTP instrumentation to ensure data accuracy
+* Decouple map tagging with uploads in plugin to improve the build pipeline by making the tagged map a cachable build artifact
+
+
+## Fixed
+
+* Fix NoClassDefFoundError when using agent v7.0.0 due to SHA signed libraries are not instrumented
```

Comment:
How about, "
* Fix NoClassDefFoundError when using agent v7.0.0 due to uninstrumented signed libraries"?